pkr/utils.py

```diff
@@ -294,11 +294,14 @@ def copy(self, path, origin, local_dst, excluded_paths, gen_template=True):
                 # os.chmod(str(dst_path), 0o600) # make invisible to the world
             else:
                 shutil.copy2(path_str, str(dst_path))
-        else:
+        elif path.is_dir():
```

Comment:
Since the above changes means we add the Dockerfile-name-matching folder explicitly now (instead of optionally, via a glob), I added this check in case the folder doesn't exist.